### ISSUE TITLE
Bug: t<char>  followed by ; does not go to the next match

### ISSUE THREAD

**Issue by keithn** [jaredpar/VsVim#2389]
Version: 2.6.1226

given the text 

```
 a b c a b c 
```
```tb```   followed by a ``` ;```    will not take you just before the second b

**Comment by ricksladkey** [comment]
Indeed, it does not. It is fascinating that such an "obvious" bug can exist for so long! The bug is not in the repeating because `tb` followed by `tb` exhibits the same symptom. The bug is that the `t<char>` command in vim always moves the caret, similar to how `/<pattern>`  moves the caret when the pattern matches immediately.

**Comment by ricksladkey** [comment]
Retesting, I got that wrong, the problem *is* in the repeating, i.e. it is in the `;` command, which always moves, whereas `t<char>` does its condition is already satisfied.

**Comment by ricksladkey** [comment]
VsVim doesn't support the vim `cpo` variable (yet) but `:help cpo-;` in vim shows that VsVim is behaving the way that the original vi behaved and that the vim behavior is a turned on by default:

```
		;	When using |,| or |;| to repeat the last |t| search
			and the cursor is right in front of the searched
			character, the cursor won't move. When not included,
			the cursor would skip over it and jump to the
			following occurrence.
```

**Comment by keithn** [comment]
fantastic!
